public/docs/ts/latest/guide/testing.jade

```diff
@@ -229,30 +246,31 @@ a#1st-karma-test
 :marked
   **Put spec files somewhere within the `src/app/` folder.**
   The `karma.conf.js` tells karma to look for spec files there,
-  for reasons explained [below](#q-spec-file-location).
+  for reasons explained [later](#q-spec-file-location).
```

Comment:
You made me laugh. I changed it back. Below sounds more natural and lets you know it's in the same page.